README.md

```diff
@@ -1,213 +1,158 @@
 # Zipline
 
 This library streamlines using Kotlin/JS libraries from Kotlin/JVM and Kotlin/Native programs. It
-makes it possible to do continuous deployment of mobile apps, just like we do for servers and web
+makes it possible to do continuous deployment of mobile code, just like we do for servers and web
 applications.
```

Comment:
I would feel better if the second sentence was phrased as one possible use case rather than the only use case. Continuous deployment is only one use case among things like user scripts, plugin systems, etc. and mobile is only one domain where you could easily run this in desktop app, servers, games, etc. Don't want to pigeon hole the project.

Reply:
Done.

-----

This library streamlines using Kotlin/JS libraries from Kotlin/JVM and Kotlin/Native programs. It
makes it fetching code as easy as fetching data:

 * For continuous deployment within mobile apps, just like we do for servers and web apps.
 * For user-customizable behavior and plugin systems
 * For updating business rules, like pricing or payments
 * For fresh content like games

It'd be simpler to do continuous deploys via the App Store & Play Store! But that process is too
slow. Even if we could release via these channels every day, we can’t guarantee that user’s devices
will take the updates immediately. For Cash App we support apps that haven’t been updated for a full
year.